integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -682,6 +716,16 @@ private BlockMasterInfo acquireBlockMasterInfo() {
     }
   }
 
+  @Nullable
+  private Map<String, MountPointInfo> acquireMountPointInfo() {
+    try {
+      return mFileSystem.getMountTable();
+    } catch (IOException | AlluxioException e) {
+      LOG.error("Failed to retrieve mount point information", e);
+      return null;
```

Comment:
> Does Suppliers.memoizeWithExpiration handle null well? how it's handled?

It seems just get null 
`
@Override
    public T get() {
      // Another variant of Double Checked Locking.
      //
      // We use two volatile reads. We could reduce this to one by
      // putting our fields into a holder class, but (at least on x86)
      // the extra memory consumption and indirection are more
      // expensive than the extra volatile reads.
      long nanos = expirationNanos;
      long now = Platform.systemNanoTime();
      if (nanos == 0 || now - nanos >= 0) {
        synchronized (this) {
          if (nanos == expirationNanos) { // recheck for lost race
            T t = delegate.get();
            value = t;
            nanos = now + durationNanos;
            // In the very unlikely event that nanos is 0, set it to 1;
            // no one will notice 1 ns of tardiness.
            expirationNanos = (nanos == 0) ? 1 : nanos;
            return t;
          }
        }
      }
      return value;
    }
`


And I refer method mFsStatCache, it get null too.
And I think get null better compared with am empty map because of it indicates an error. The program should not go any further here.